app/org/sagebionetworks/bridge/dynamodb/DynamoSubpopulationDao.java

```diff
@@ -104,15 +112,17 @@ public Subpopulation createDefaultSubpopulation(StudyIdentifier studyId) {
         subpop.setName("Default Consent Group");
         subpop.setMinAppVersion(0);
         subpop.setDefaultGroup(true);
+        // The first group is required until the study designers say otherwise
+        subpop.setRequired(true); 
```

Comment:
Added checks that it can be set/changed in CRUD operations, and that it's set to true when a default subpopulation is created.